guestbook/redis-follower/run.sh

```diff
@@ -1,7 +1,25 @@
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_redis_follower_run]
```

Comment:
does this region tag follow the directory / resource name convention for naming region tags?

Reply:
For `.sh` files, we have it as <product-prefix>_<directory-with-underscores>_<filename-without-extension>
let me know if I should change it!